index.html

```diff
@@ -1,11 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="index.css" />
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Bio Page</title>
 </head>
-<body>
-    
+<body bgcolor="beige">
+    <div align="center">
+        <img id="img" src="image/anu.jpeg">
+        <br>
+        <h2 style="text-align:center;line-height: 5px;">Anuradha Sathiyachalam</h2>
+        <h3 style="text-align:center;line-height: 5px;">Dallas, TX</h3>
+        <h3 style="text-align:center;line-height: 5px;">Email: anoosathyee@gmail.com</h3>
+
+
+
+         <a href="https://www.linkedin.com/in/anuradha-sathiyachalam-9aa49265/"
+                target="_blank">LinkedIn</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="https://github.com/AnooSathyee"
+                target="_blank">GitHub</a>
+        <hr>
+    </div>
+    <ul>
+        <li>I am a fun loving person</li>
+        <li>Love to learn new technologies</li>
+        <li>Adapt things really fast</li>
+        <li>I love <b>Coffee</b></li>
+        <li>My favorite color is <span style="color: #ddf50a">Yellow</span></li>
+        <li>I love traveling</li>
+        <li>Looking for a change in career from non IT to IT...</li>
+    </ul>
+    <hr>
+    <a href="gallery.html">My Favorites!</a><br><br>
+    <a href="contact.html">To Reach Me!</a>
+    <br><br>
+    <b>Thank you TEKsystems for giving me this opportunity!</br>
```

Comment:
There is an orphaned 'b' tag, which doesn't have a matching '/b' tag.  Was this in error?  Is the '/br' tag at the end of the line supposed to be the '/b' tag?